packages/v4/patternfly-docs/components-data.json

```diff
@@ -0,0 +1,354 @@
+{
+  "about-modal": {
+    "illustration": "./images/component-illustrations/./images/component-illustrations/about-modal.png",
+    "summary": "An <b>about modal</b> displays information about an application like product version number(s), as well as any appropriate legal text."
+  },
+  "accordion": {
+    "illustration": "./images/component-illustrations/accordion.png",
+    "summary": "An <b>accordion</b> is an interactive container that expands and collapses to hide or reveal nested content. It takes advantage of progressive disclosure to help reduce page scrolling, by allowing users to choose whether they want to show or hide more detailed information as needed."
+  },
+  "action-list": {
+    "illustration": "./images/component-illustrations/action-list.png",
+    "summary": "An <b>action list</b> is a group of actions with set spacing."
+  },
+  "alert-group": {
+    "illustration": "./images/component-illustrations/alert-group.png",
+    "summary": "An <b>alert group</b> stacks and positions 2 or more alerts, either in a layer over the main content of a page or inline with the page content. Alert groups should always rank alerts by age, stacking new alerts on top of old ones as they surface."
```

Comment:
Functionally, another primary use case for Alert Groups is for creating a live region for alerts. Maybe the first sentence could say, `An alert group stacks and positions 2 or more alerts in a live region, either in a layer over the main content of a page or inline with the page content...`